protobuf/concept.proto

```diff
@@ -615,7 +616,10 @@ message Relation {
     message Players {
         message Iter {
             message Req {
-                repeated Concept roles = 1;
+                message All {}
+                message ForRoles {
+                    repeated Concept roles = 1;
+                }
```

Comment:
Every `message` inside this `message Relation` object represents a `method` on the `Relation` class in Grakn, @alexjpwalker. As you may notice `playersMap`, `relate`, and `unrelate` are all methods on the `Relation` class. Given that the goal of this PR is to replace `players` method with 2 different methods, we should be replacing the `message Players` object altogether with 2 new ones.

Reply:
The 2 different methods will both be named `players`. What should we do in this case?

Reply:
We decided to split the message up into AllPlayers  and PlayersForRoles.